### DISCUSSION TITLE
Delete project

### DISCUSSION THREAD

**Discussion by yoridewit** [AntlerVC/firetable#838] [Support / Q&A] [answered]
Hi folks,

Is there a a way to delete my project in rowy? 
I deleted my service account and firestore collection a while ago, but I changed my mind and want to try Rowy again.
I just can't seem to make the manual install work, I've tried a couple of nights in a row now.

Seems like if I can just do the default project startup again I'll be fine.

Thanks in advance,
Yori

**Comment by BeeBombshell** [comment, marked as answer]
Hey @yoridewit, as discussed in #840. In a recent update, Rowy adds the capability of personalizing the workspace and controlling its projects via Workspace settings. 

Hope this helps!

https://user-images.githubusercontent.com/53828745/198615756-4cd2886b-bc94-4353-92e6-969a2a0242ca.mp4

**Reply by yoridewit** [reply]
Thank you!
